app/services/post_service.rb

```diff
@@ -21,6 +21,23 @@ def find!(id_or_guid)
     end
   end
 
+  def present_api_json
+    service = CommentService.new(post_id: post.id, user: user)
```

Comment:
There is no such method `post` in current class which again points that this code is not covered by specs.

Reply:
Thx for the review, but I didn't have clean that one yet. I will handle it when I will do it.